internal/config/config.go

```diff
@@ -31,6 +31,8 @@ type OolongConfig struct {
 	PluginsConfig OolongPluginConfig `toml:"plugins"`
 	GraphConfig   OolongGraphConfig  `toml:"graph"`
 	SyncConfig    OolongSyncConfig   `toml:"sync"`
+
+	PinningEnabled *bool `toml:"pinning_enabled"`
```

Comment:
`pinning_enabled` can just be a `bool`, it will default to false if the value isn't present in the config file.

Reply:
that would also simplify the functions and checks related to that variable as well